src/Components/Link/style.scss

```diff
@@ -0,0 +1,31 @@
+@import './../../vars';
+
+$link-color: #7955ff;
+$link-visited-color: #b5a1ff;
+$link-hover-color: #4729b3;
+$transition-duration-short: 0.5s;
+$transition-duration-long: 1s;
+
+.link-of-trust {
+    margin: $padding;
+    font-family: $default-font;
+}
+
+.link-of-trust:link {
+    color: $link-color;
+    text-decoration: none;
+    -webkit-transition: color $transition-duration-short;
```

Comment:
This is not needed in most cases. We should add autoprifixer to build process instead when it will be needed.

Reply:
Remove this line and we are good with this PR

Reply:
which line u mean exactly? or you mean the rule itself?